src/cli.rs

```diff
@@ -299,6 +299,7 @@ pub async fn main<H: Hooks, M: MigratorTrait>() -> eyre::Result<()> {
             let environment = Environment::from_str(&environment)
                 .unwrap_or_else(|_| Environment::Any(environment.to_string()))
                 .load()?;
+            logger::init::<H>(&environment.logger);
```

Comment:
is there a chance we need to rethink logger init? 
like move this to the top right after main(), and put it only there (remove all init from context creation, etc)

this might be also the same pain point with logs in tests?

Reply:
i think we need to init the logger in the cli.rs file